CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 3.1.1 (Under development)
```

Comment:
```suggestion
## Version 3.2.0 (Under development)
```

Since it's labeled as feature.
Should also bump versions.gradle to use 3.2 so snapshot builds are correctly versioned as well.